designs/Verify-Verbosity.md

```diff
@@ -0,0 +1,410 @@
+
+## Problem background
+
+`dotnet nuget verify` command doesn't provide any output by default due to logical error in the implementation. If the user didn't pass value for `--verbosity` option, then the log level is set to [`LogLevel.Minimal`](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Signing/VerifyCommand.cs#L58). Customers need information about packages’ signatures to [manage package trust boundaries](https://docs.microsoft.com/en-us/nuget/consume-packages/installing-signed-packages).
+
+## Who are the customers
+
+Package consumers that use the `dotnet nuget verify` command to verify package signature and configure NuGet security policy.
+
+## Goals
+
+* Change the output of `dotnet nuget verify` command for various `--verbosity` options.
+* Improve log messages & error codes in docs to improve the customer experience based upon the learnings from the recent Debian incident.
+
+## Non-goals
+
+* Add `--verbosity` option to `dotnet nuget verify` command.
+
+## Solution
+
+The details that should be displayed on each verbosity level are described below. Each level should display the same as the level below plus whatever is specified in that level. In that sense, `quiet` will be give the least amount of information, while `diagnostic` the most.
+
+​                                  | `q[uiet]` | `m[inimal]` | `n[ormal]` | `d[etailed]` | `diag[nostic]`
+----------------------------------| --------- | ----------- | ---------- | -----------| --------------
+`Certificate chain Information`   | ❌       | ❌          | ❌         | ✔️         | ✔️
+`Path to package being verified`  | ❌       | ❌          | ✔️         | ✔️         | ✔️
+`Hashing algorithm used for signature`        | ❌       | ❌          | ✔️         | ✔️         | ✔️
+`Author/Repository Certificate -> SHA1 hash`| ❌       | ❌          | ✔️         | ✔️         | ✔️
+`Author/Repository Certificate -> Issued By`| ❌       | ❌          | ✔️         | ✔️         | ✔️
+`Timestamp Certificate -> Issued By`| ❌       | ❌          | ✔️         | ✔️         | ✔️
+`Timestamp Certificate -> SHA-256 hash`| ❌       | ❌          | ✔️         | ✔️         | ✔️
+`Timestamp Certificate -> Validity period`| ❌       | ❌          | ✔️         | ✔️         | ✔️
+`Timestamp Certificate -> SHA1 hash`| ❌       | ❌          | ✔️         | ✔️         | ✔️
+`Timestamp Certificate -> Subject name`| ❌       | ❌          | ✔️         | ✔️         | ✔️
+`Author/Repository Certificate -> Subject name`| ❌       | ✔️          | ✔️         | ✔️         | ✔️
+`Author/Repository Certificate -> SHA-256 hash`| ❌       | ✔️          | ✔️         | ✔️         | ✔️
+`Author/Repository Certificate -> Validity period`| ❌       | ✔️          | ✔️         | ✔️         | ✔️
+`Author/Repository Certificate -> Service index URL (If applicable)`| ❌       | ✔️          | ✔️         | ✔️         | ✔️
```

Comment:
This should be:  Repository Certificate -> Service index URL

Author certificates won't have a service index URL and repository certificates MUST have this attribute; there's no "if applicable" --- it's always applicable.